src/commands/server.cpp

```diff
@@ -22,6 +22,61 @@
 #include <vector>
 
 volatile std::sig_atomic_t eoserv_sig_abort = false;
+volatile std::sig_atomic_t eoserv_sig_reload = false;
+
+TimeEvent* shutdown_timer = nullptr;
+
+static void schedule_shutdown(const std::vector<std::string>& arguments, Command_Source* from, bool is_shutdown)
```

Comment:
`abort` doesn't feel right because the process is still running in the case of a reload. I think the term shutdown is fine because a reload is a shutdown of sorts, but I agree it's confusing to pass a parameter `is_shutdown` to a function `schedule_shutdown`. I'll see if I can come up with better names.

Reply:
Isn't `is_shutdown` the parameter that determines whether the application will "shut down", AKA abort?
I just snagged the word `abort` from the corresponding `eoserv_sig_abort`.

Reply:
I'm going with naming the parameter `is_reload` and inverting the logic that checks it, I think that makes it a little clearer